src/testing.md

```diff
@@ -0,0 +1,88 @@
+# Testing
+
```

Comment:
Again, another paragraph for what to expect on this page would be nice. Again, doesn't need to be in this PR.